R/carehomes_rt.R

```diff
@@ -47,7 +49,8 @@ carehomes_Rt <- function(step, S, p) {
       mean_duration_weighted <- drop(mean_duration[, , t])
     }
 
-    ngm <- outer(mean_duration_weighted, S_weighted) * m
+    ## In a multistrain model R0 is the max of R0 across strains
+    ngm <- outer(mean_duration_weighted, S_weighted) * m * max_strain_multiplier
```

Comment:
Can you open a [new issue](https://github.com/mrc-ide/sircovid/issues/new) to document what needs doing to make this robust

Reply:
new issue here: https://github.com/mrc-ide/sircovid/issues/184